#req

int8 MODE_GRIPPER_PIVOTING=1
int8 MODE_SLIPPING_AVOIDANCE=2
int8 MODE_DYN_SLIPPING_AVOIDANCE=3

int8 mode
---
#res
bool success
int8 state
string msg
---
#fbk